### PR TITLE
Allow JSON attributes

### PR DESCRIPTION
**Pull request by tbremer**

### CODE REVIEW

**Comment by RyanHirsch** [comment]
This fixes the short term issue with stringified JSON, but doesn't actually solve the problem. If there is a single quote in the payload, it breaks too.

File: server/renderer/index.js [hunk 1]
```diff
@@ -7,8 +7,14 @@ function createNode(name) {
...
-
-  return `${key}="${value === true ? key : value}"`;
+  if (value === true) return `${key}="${key}"`;
+  const quoteStyle = /"/.test(value) ? "'" : '"';
+  let attr = `${key}=`;
+  attr += quoteStyle;
+  attr += value;
+  attr += quoteStyle;
+
+  return attr;
```
Comment (RyanHirsch) [inline server/renderer/index.js]:
```suggestion
  return `${key}="${value === true ? key : value.replace(/"/g, '&quot;')}"`;
```

Comment (RyanHirsch) [reply server/renderer/index.js]: This seems to fix the problem. I can then `getAttribute` and `JSON.parse` the value

Comment (tbremer) [reply server/renderer/index.js]: Yeah, yeah, be thorough. ;) This looks good and is similar to how `ReactDOMServer.renderToString` handles attributes. I didn't see a case where we had `"` and `'` in a single JSON blob, so I didn't worry about it. 😂

**Pull request merged**